### PR TITLE
onboarding and backup key widgets

### PR DESCRIPTION
**Pull request by Sameeraja-Bharathula**

### CODE REVIEW

**Comment by murali-shris** [comment]
please remove commented code and print statements

File: at_backupkey_flutter/example/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  flutter:
+    sdk: flutter
+
+  at_backupkey_flutter:
+    # When depending on this package from a real application you should use:
+    #   at_backupkey_flutter: ^x.y.z
+    # See https://dart.dev/tools/pub/dependencies#version-constraints
+    # The example app is bundled with the plugin so we use a path dependency on
+    # the parent directory to use the current plugin's version.
+    path: ../
```
Comment (murali-shris) [inline at_backupkey_flutter/example/pubspec.yaml:21]: is this relative path correct?

Comment (Sameeraja-Bharathula) [reply at_backupkey_flutter/example/pubspec.yaml:21]: yes.

File: at_onboarding_flutter/lib/utils/app_constants.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import 'package:at_client_mobile/at_client_mobile.dart';
+
+class AppConstants {
+  static String _rootDomain = 'root.atsign.org';
```
Comment (murali-shris) [inline at_onboarding_flutter/lib/utils/app_constants.dart:4]: can we move this to config ?

Comment (Sameeraja-Bharathula) [reply at_onboarding_flutter/lib/utils/app_constants.dart:4]: will take it up in the subsequent PR.

**Pull request merged**